Build/Projects/FrameworkReferences.definition

```diff
@@ -8,6 +8,9 @@
     <Reference Include="OpenTK-1.0" />
   </Platform>
   <Platform Type="tvOS">
+    <Reference Include="System" />
+    <Reference Include="System.ComponentModel.Composition” />
```

Comment:
You have one of those fancy quotes at the end of this `”`...  needs to be a plain quote char.